### PR TITLE
 #159738422 Shops Review and Rating

### PR DESCRIPTION
**Pull request by bekomay26**
#### What does this PR do?
- Adds customer reviews and ratings for shops.

#### Description of Task to be completed?
Customers should be able to rate and review a  store. When a user visits a store page, the user should be able to rate the store and add a review of the store

#### How should this be manually tested?
Navigate to the store page and click on the View reviews button

#### What are the relevant pivotal tracker stories?
159738422

#### Any background context you want to add?
N/A

Screenshot

image
image

### CODE REVIEW

**Comment by Nkemjiks** [comment]
Hi @bekomay26 and @Hasstrup, please include a complete screenshot of the whole page so we can see the shop as well. Also do well to resolve all hound issues.

**Review by Nkemjiks — CHANGES_REQUESTED** [review]
Please check the comment and respond accordingly

**Comment by bekomay26** [comment]
@Nkemjiks Thanks for the review, I have changed the screenshots and I am fixing the hound issues.

File: imports/plugins/custom/shop-review/server/publications/review.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { Meteor } from "meteor/meteor";
+import { check } from "meteor/check";
+// import { Reviews } from "../../../reviews/lib/collections";
+import { Reviews } from "../../lib/collections";
+import { Products, Shops } from "../../../../../../lib/collections";
+
+Meteor.publish("shop.reviews", function (shopId) {
```
[the remark this answers is not in the document]
Comment (bekomay26) [reply imports/plugins/custom/shop-review/server/publications/review.js]: Meteor publication doesn't work properly with arrow functions

**Comment by bekomay26** [comment]
@Nkemjiks I have fixed the hound issues

**Comment by kenware** [comment]
Nice work @bekomay26 and @Hasstrup,  however, it will be better if the list of reviews is paginated instead of loading all to the client. Checkout react-easy-pagination component for pagination

**Comment by bekomay26** [comment]
Thank you for the review @kenware, we will endeavour to implement the pagination. Also, thanks for the recommendation, it will make our work a tad bit easier.

**Review by Nkemjiks — APPROVED** [review]

**Review by Onnassiz — APPROVED** [review]
Nice job @kenware and @ikenjoku. Keep it up.

**Pull request closed without merging**